projects/portal/src/app/views/nft-pawnshop/lenders/lenders.component.html

```diff
@@ -64,25 +67,52 @@ <h2 class="card-title break-all">Bidding Status</h2>
 <div class="card lg:card-side bg-base-100 shadow-xl mb-8">
   <div class="card-body">
     <h2 class="card-title break-all">Lend Tokens</h2>
-    <form #formRef="ngForm" (submit)="onSubmit()">
+    <div class="form-control">
+      <label class="cursor-pointer label">
+        <span class="label-text">Advanced Search</span>
+        <input
+          type="checkbox"
+          class="toggle toggle-primary"
+          name="openSearch"
+          [(ngModel)]="isOpenedSearch"
```

Comment:
open search is not good word I think.
how about
"isNotFiltered"

Reply:
This is only to open the input screen.

Reply:
Understood but that variable naming is not good.